test/SlowTests/RavenDB-14303.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.ServerWide;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server.Documents.Indexes;
+using Raven.Server.Documents.Indexes.MapReduce.Auto;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Commands.Indexes;
+using Raven.Tests.Core.Utils.Entities;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests
+{
+    public class RavenDB_14303 : ClusterTestBase
+    {
+
+        [Fact]
+        public async Task IgnoreIdleIndexOnPromotable()
+        {
+            var clusterSize = 3;
+            var (_, leader) = await CreateRaftCluster(clusterSize);
+            var replicationFactor = 2;
+            var databaseName = GetDatabaseName();
+            using (var store = new DocumentStore()
```

Comment:
use something like this:
```
using (GetDocumentStore(new Options
            {
                Server = cluster.Leader,
                ReplicationFactor = 2,

            }))
```

Reply:
then you don't need stuff like `await dbServer.ServerStore.Cluster.WaitForIndexNotification(createRes.RaftCommandIndex);`

Reply:
done